Shoko.Server/Providers/AniDB/UDP/User/ResponseMyListFile.cs

```diff
@@ -2,11 +2,12 @@
 
 namespace Shoko.Server.Providers.AniDB.UDP.User
 {
-    public class ResponseAddFile
+    public class ResponseMyListFile
     {
         public bool IsWatched { get; set; }
         public DateTime? WatchedDate { get; set; }
         public GetFile_State? State { get; set; }
         public int MyListID { get; set; }
+        public DateTime UpdatedAt { get; set; }
```

Comment:
I would make that DateTime?. It can be missing, and we don't want to throw an error on that